### PR TITLE
wio terminal: Implement SD card SPI initialization + example

### PR DESCRIPTION
**Pull request by twitchyliquid64**

### CODE REVIEW

File: boards/wio_terminal/src/storage.rs [hunk 1]
```diff
@@ -42,3 +47,62 @@ pub struct SDCard {
...
+        //
+        // // Configure the backlight pin as a push-pull output; unfortunately this pin
+        // // does not appear to support PWM.
+        // //   HIGH - backlight enabled
+        // //   LOW  - backlight disabled
+        // let mut backlight = self.backlight.into_push_pull_output(port);
+        // backlight.set_high()?;
+
+        // Return a result consisting of a Tuple containing the display driver and
+        // backlight pin.
```
Comment (jessebraham) [inline boards/wio_terminal/src/storage.rs:105]:
```suggestion
```
I don't think this needs to be included 😉

File: boards/wio_terminal/Cargo.toml [hunk 2]
```diff
@@ -49,6 +53,7 @@ default = ["atsamd-hal/samd51p", "rt", "unproven", "usb"]
 rt = ["atsamd-hal/samd51p-rt", "cortex-m-rt"]
 unproven = ["atsamd-hal/unproven"]
 usb = ["atsamd-hal/usb", "usb-device", "usbd-serial"]
+sdcard = ["embedded-sdmmc"]
```
Comment (jessebraham) [inline boards/wio_terminal/Cargo.toml:56]: Is there any reason `sdcard` shouldn't be a default feature?

**Review by jessebraham — COMMENTED** [review]
Awesome, thanks!

It seems to be working as intended on my device. I just commented on a couple of little things then I think it should be good.

File: boards/wio_terminal/src/storage.rs (hunk 1 above)
Comment (twitchyliquid64) [reply boards/wio_terminal/src/storage.rs]: Oooof good catch!

File: boards/wio_terminal/Cargo.toml (hunk 2 above)
Comment (twitchyliquid64) [reply boards/wio_terminal/Cargo.toml]: No strong inclination - Ive removed that feature as its not really a biggie for it to be a dep.

**Review by jessebraham — APPROVED** [review]
Thank you for making the requested changes, and for the additional changes you made as well!

**Pull request merged**